data/Viveksati5143.json

```diff
@@ -1,16 +1,61 @@
 {
   "name": "Vivek Sati",
-  "bio": " **Open Source**. Devops enthusiast",
+  "bio": " **Open Source** | DevOps Intern @Incloudo ",
+  "tags": [
+    "Open Source",
+    "DevOps",
+    "AWS"
+  ],
+  "socials": [
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/Vivek__twts"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/Viveksati5143"
+    },
+    {
+      "icon": "SiLinkedin",
+      "url": "https://www.linkedin.com/in/vivek-sati"
+    }
+  ],
+  "testimonials": [
+    
+  ],
```

Comment:
You can remove this section because its empty.

Reply:
Then how will someone add testimonials?